sandbox-web/src/main/java/com/exadel/sandbox/team5/OrderRestController.java

```diff
@@ -41,9 +42,9 @@ public void delete(@PathVariable Long id) {
         orderService.delete(id);
     }
 
-    @GetMapping("/invalidate")
-    public OrderDto invalidate(@PathVariable Long discountId, @PathVariable String promoCode) {
-        return orderService.invalidatePromoCode(discountId, promoCode);
+    @PutMapping("/invalidate")
```

Comment:
Just a question can we use order id from frontentend?

Reply:
Method is refactored with only one input parametr promoCode.

Reply:
I think discountId is redundant field.